src/nodekit/NKScripting/util/NKLogging.swift

```diff
@@ -174,29 +173,29 @@ public class NKLogging {
         
         #if DEBUG
             
-            filter = Filter(upto: .Debug)
+            filter = Filter(upto: .debug)
             
         #else
             
-            filter = Filter(upto: .Notice)
+            filter = Filter(upto: .notice)
             
         #endif
         
         let format = format ?? "$((Time)(lcl)) $(Facility) <$((Level)(char))>: $(Message)"
         
-        asl_add_output_file(client, STDERR_FILENO, format, "sec", filter.rawValue, 1)
+        _ = asl_add_output_file(client, STDERR_FILENO, format, "sec", filter.rawValue, 1)
```

Comment:
Is there no way to annotate this method with `@discardableResult`?  I'm not familiar with the syntax at the top of this file so perhaps it's not possible.

Reply:
I am not sure, we're using this _sil_gen_name keyword to declare these functions and I think they are linked at runtime, so I think I have to leave the signature how it is. Should be fine either way.